be/src/olap/schema.h

```diff
@@ -78,16 +98,22 @@ class Schema {
 
     ~Schema();
 
-    void reset(const std::vector<Field>& cols, size_t num_key_columns);
+    void reset_field(const std::vector<TabletColumn>& columns,
```

Comment:
you should write some comments for reset_field and reset function. And I think these functions should be as private. If we want let people reset this schema, we should provide another function wrapper these two functions.

Reply:
OK